eventhubs/client/azure-eventhubs/src/main/java/com/azure/eventhubs/implementation/AmqpException.java

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.eventhubs.implementation;
+
+import org.apache.qpid.proton.amqp.transport.ErrorCondition;
+
+/**
+ * All AmqpExceptions - which EventHub client handles internally.
+ */
+public class AmqpException extends Exception {
```

Comment:
File an issue to elevate these exceptions up into the azure-core module.....without having any apache-qpid dependency :-)